nbs/_quarto.yml

```diff
@@ -4,17 +4,51 @@ project:
 format:
   html:
     theme: cosmo
+    fontsize: 1em
+    linestretch: 1.7
     css: styles.css
     toc: true
 
 website:
   twitter-card: true
   open-graph: true
+  google-analytics: "G-NXJNCVR18L"
   repo-actions: [issue]
+  favicon: favicon_png.png
   navbar:
     background: primary
     search: true
+    collapse-below: lg
+    left:
+      - text: "Get Started"
+        href: examples/Getting_Started_short.ipynb
+      - text: "NixtlaVerse"
+        menu:
+          - text: "MLForecast 🤖"
+            href: https://github.com/nixtla/mlforecast
+          - text: "NeuralForecast 🧠"
+            href: https://github.com/nixtla/neuralforecast
+          - text: "StatsForecast ⚡️"
+            href: https://github.com/nixtla/neuralforecast
+          
+      - text: "Help"
+        menu:
+          - text: "Report an Issue"
+            icon: bug
+            href: https://github.com/nixtla/hierarchicalforecast/issues/new/choose
+          - text: "Join our Slack"
+            icon: chat-right-text
+            href: https://join.slack.com/t/nixtlacommunity/shared_invite/zt-1kd5m5db7-7OOfy0xVNf1PcvCiAPWvPw
+    right:
+      - icon: github
+        href: "https://github.com/nixtla/hierachicalforecast"
+      - icon: twitter
+        href: https://twitter.com/nixtlainc
+        aria-label: Nixtla Twitter
+
   sidebar:
     style: floating
+  body-footer: |
+    Give us a ⭐ on [Github](https://github.com/nixtla/hierachicalforecast)
```

Comment:
Can we add like a friendly "if" condition?
"If you find the code useful, please give us a star"